src/lexer/CMakeLists.txt

```diff
@@ -93,8 +93,6 @@ add_custom_command(
 add_custom_command(
   OUTPUT ${PROJECT_BINARY_DIR}/src/parser/diffeq/diffeq_parser.cpp
          ${PROJECT_BINARY_DIR}/src/parser/diffeq/diffeq_parser.hpp
-         ${PROJECT_BINARY_DIR}/src/parser/diffeq/location.hh
-         ${PROJECT_BINARY_DIR}/src/parser/diffeq/position.hh
          ${PROJECT_BINARY_DIR}/src/parser/diffeq/stack.hh
   COMMAND ${BISON_EXECUTABLE} ARGS -d -o ${PROJECT_BINARY_DIR}/src/parser/diffeq/diffeq_parser.cpp
           ${NMODL_PROJECT_SOURCE_DIR}/src/parser/diffeq.yy
```

Comment:
We can clean that by using https://cmake.org/cmake/help/v3.8/module/FindBISON.html

Reply:
Do you mean using `BISON_TARGET`?

Reply:
Yes

Reply:
notice that `FLEX_TARGET` exists too

Reply:
@alkino : I looked at BISON_TARGET and it has form:

> BISON_TARGET(MyParser parser.y parser.cpp DEFINES_FILE parser.h)

what I am missing is that possibility to set dependency with other header files. e.g. In our case `diffeq.yy` depends on `diffeq_context.hpp` and `diffeq_helper.hpp`. I don't see straightforward/compact way to add extra dependencies to `MyParser` target.

So, I think its fine to keep the current custom_command.